gradle.properties

```diff
@@ -10,7 +10,7 @@ evosuiteVersion = 1.0.5
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild = 241
+pluginSinceBuild = 242
```

Comment:
Does this mean that we lose the audience that has not yet transitioned to newer IntelliJ version?

@pderakhshanfar